leeanp.php

```diff
@@ -24,3 +24,6 @@
 // Run the plugin setup.
 $setup = new PluginSetup();
 register_activation_hook( __FILE__, [ $setup, 'maybe_deactivate' ] );
+
+// Init ECommerce endpoints.
+add_action( 'rest_api_init', [ $setup, 'init' ] );
```

Comment:
OK I see can we remove this instead then.

https://github.com/moxie-lean/wp-woocommerce-api/blob/395c6a04c18011778372bcdff5c25d333409e2f7/PluginSetup.php#L16-L21

And just replace with: 

```php
add_action( 'rest_api_init', [ Bootstrap, 'init' ] );
```